automation_tools/__init__.py

```diff
@@ -2153,12 +2190,24 @@ def satellite6_upgrade(admin_password=None):
     run('yum clean all', warn_only=True)
     # Updating the packages again after setting sat6 repo
     print('Wait till packages update ... ')
+    print('YUM UPDATE started at: {0}'.format(time.ctime()))
     update_packages(quiet=True)
+    print('YUM UPDATE finished at: {0}'.format(time.ctime()))
+    # Rebooting the system again for possible errors
+    reboot(env.get('satellite_host'), 120)
```

Comment:
better to call execute here and change the reboot to a plain task.
 Add a line note